styles.css

```diff
@@ -381,5 +384,102 @@ main {
   display: flex;
   list-style: none;
   padding-left: 0;
-  gap: 235px;
+  gap: 255px;
+}
+
+.about-padding-left {
+  padding-left: 24px;
+}
+
+.contact-form {
+  padding-left: 24px;
+  width: 375px;
+  height: 779px;
+  float: center;
+  margin-top: 100px;
+  padding-top: 120px;
+  background: #6070ff;
+  border-top-left-radius: 80px;
+  background-image: url("images/contact-form-shapes-mobile.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+}
+
+.contact-form-details {
+  display: flex;
+  flex-direction: column;
+  justify-content: right;
+  align-items: right;
+  row-gap: 12px;
+}
+
+.contact-title {
+  margin-top: 50px;
+  margin-bottom: 0;
+  width: 801px;
+  height: 56px;
+  left: -213px;
+  top: 4734px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  padding-left: 50px;
+  color: #fff;
+}
+
+.contact-details {
+  width: 327px;
+  height: 102px;
+  left: 24px;
+  top: 4802px;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact-form-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  row-gap: 20px;
+  padding-left: 0;
+  margin-top: 77px;
+}
+
+input {
+  width: 330px;
+  height: 40px;
+  border-radius: 8px;
+}
+
+textarea {
+  width: 330px;
+  height: 170px;
+  border-radius: 8px;
+}
+
+.contact-form-button {
+  padding: 12px 16px;
+  width: 140px;
+  height: 48px;
+  left: 24px;
+  background: #fff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+}
```

Comment:
- [ ] You did a wonderful job styling the submit button :+1: , but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), you should follow the same design as the Figma template and add some button interactions such as `hover`, and `active` (when it's pressed), as shown in the Figma template :+1:.

![Screenshot from 2023-01-14 07-40-42](https://user-images.githubusercontent.com/105117832/214517055-2183303b-a38c-4a23-bce9-06bb71f7f124.png)